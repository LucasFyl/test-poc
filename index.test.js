import React from 'react';
import StyledTemplate from './src';
import renderer from 'react-test-renderer';

test('Render Template component', () => {
    const component = renderer.create(
        <StyledTemplate></StyledTemplate>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
