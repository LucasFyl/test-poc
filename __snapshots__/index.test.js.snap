// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Template component active 1`] = `
<div
  className="default_msu5iu-o_O-red_16geqrx-o_O-hover_djbono-o_O-responsive_1fuov80"
/>
`;

exports[`Render Template component not active 1`] = `
<div
  className="default_msu5iu-o_O-blue_c4lqni-o_O-hover_djbono-o_O-responsive_1fuov80"
/>
`;
